pages/503.js

```diff
@@ -0,0 +1,3 @@
+export default function Error503() {
+  return <h1>503 - Service unavailable error occurred</h1>
```

Comment:
Same here!

Reply:
done in b18b6c06469fc4da56ab4cde1926e2e1275d6e73